README.md

```diff
@@ -871,7 +871,7 @@ When documenting an unstable feature, those unstable packages must be explicitly
 <PackageReference Include="NServiceBus.RabbitMQ" Version="4.3.1-alpha.XYZ" />
 ```
 
-Wildcard patterns, like `4.3.1-*`, or `8.0.0-alphe/beta/rc.*` are not allowed for pre-releases since they tend to break the snippet and sample builds.
+Wildcard patterns, like `4.3.1-*`, `8.0.0-alpha.*`, `8.0.0-beta.*`, or  `8.0.0-rc.*` are not allowed for pre-releases since they tend to break the snippet and sample builds.
```

Comment:
```suggestion
Wildcard patterns, like `4.3.1-*`, `8.0.0-any-pre-release-version.*` are not allowed for pre-releases since they tend to break the snippet and sample builds.
```

SemVer calls alpha, beta, rc etc. simply pre-release-version if I'm not mistaken https://semver.org/